index.html

```diff
@@ -1615,18 +1615,57 @@ <h3>Media Types</h3>
 	<section id="sec-building-blocks">
 		<h1>WoT Building Blocks</h1>
 		<p>
-			The Web of Things provides more concrete building blocks that allow to implement the abstract WoT Architecture.
-			As, they are defined in separate documents, this document provides the overview and a summary.
+			The Web of Things provides specific building blocks to allow the implementation of systems that
+                        conform with the abstract WoT Architecture.
+			The specfics of these building blocks are defined in separate documents; 
+                        this document provides an overview and a summary.
 		</p>
-		
-		<p>
 			<figure id="arch-new-building-blocks">
-				<img src="images/arch-new-building-blocks.png" />
-				<figcaption>Building Blocks</figcaption>
+				<img src="images/wot-building-blocks.png" />
+				<figcaption>Architectural Aspects of a Thing</figcaption>
+			</figure>
+		<p>
+                        The use cases and the requirements discussed result in an abstract model of 
+                        a Thing that has four architectural aspects of interest: its behavior, its interaction affordances,
+                        the mapping of those affordances to concrete protocols (what we will call the protocol binding),
+                        and its security configuration.  
+                        The behavior aspect includes both lifecycle management (onboarding, updating, decommissioning, etc) but
+                        also the operational behavior of the Thing, which can include both autonomous behavior and support 
+                        for the affordances.  The interaction affordances are an abstract model of the network API in terms
+                        of properties, actions, and events, but without reference to a specific network protocol or
+                        payload encoding.  The protocol binding adds the additional detail needed to map each
+                        affordance to a particular concrete protocol; not that different affordances may be supported
+                        by different protocols.  The security configuration aspect represents the mechanisms used to
```

Comment:
Please consider rewording: Node that different protocols may support different sets of affordances.

Reply:
done... I think.